Gruntfile.js

```diff
@@ -31,11 +31,10 @@ module.exports = function( grunt ) {
 		],
 		tests: {
 			"jquery": [
-				"dev+2.x-git",
-				"min+2.x-git.min",
-				"dev+2.2.2",
-				"dev+1.x-git",
-				"dev+1.12.2"
+				"dev+git",
+				"min+git.min",
+				"dev+3.0.0",
```

Comment:
This will work only after the final 3.0.0 release, won't it? We should probably put `3.0.0-beta` for now. Or wait with this entry for a final release and for now just do the `git` part.